openpathsampling/tests/test_pathmover.py

```diff
@@ -11,6 +11,7 @@
 from nose.plugins.skip import SkipTest
 from nose.tools import (assert_equal, assert_not_equal, raises, assert_true,
                         assert_in, assert_not_in)
+import numpy as np
```

Comment:
I don't see this import being used -- is this leftover from earlier tests?

Reply:
Seems like it